src/helpers_test.js

```diff
@@ -1044,5 +1044,14 @@ describe("helper functions", () => {
       const sibling = helpers.getNextSibling(originNode, ".pick")
       assert.equal(sibling.innerHTML, "Next With Selector")
     })
+
+    it("replaceIrregularWhitespace replaces irregular whitespace characters in a string with a regular space", () => {
+      const testText =
+        "String\u00a0with\u00a0unicode\u00a0non\u00a0breaking\u00a0spaces"
```

Comment:
**Suggestion:** Might be nice to test `A\fB\nC\rD\tE\vF\u00A0G\u2028H\u2029I` equals `A B C D E F G H I`. I know `\u00` was the motivation, but could be nice to check the others, if only to document it.

Reply:
Good suggestion!